### PR TITLE
review wanted

### PR DESCRIPTION
**Pull request by kajsabootin**
Link to my deployed project:

https://happy-thoughts-api-kajsa.herokuapp.com/

frondend: https://kajsa-happythought.netlify.app/

Things I'd like to have clarified or explained in more detail:

I had really problem to understand how I was able to check if "like" worked.

I am most proud of that this is my first project that is both backend and frontend. So crazy??????

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,29 +1,87 @@
...
+});
+
+//Get the 20 latest thoughts route
+app.get("/thoughts", async (req, res) => {
+  const thoughts = await Thought.find()
+    .sort({ createdAt: "desc" })
+    .limit(20)
+    .exec();
+  res.json(thoughts);
+});
```
Comment (KarolCislowski) [inline server.js:51]: according to documentation` get "/" ` need to return 20 thoughts,

File: server.js [hunk 2]
```diff
@@ -1,29 +1,87 @@
...
+  try {
+    // Succes
+    const savedThought = await thought.save();
+    res.status(201).json(savedThought);
+  } catch (err) {
+    res
+      .status(400)
+      .json({ message: "Sorry, could not post this", error: err.errors });
+  }
+});
```
Comment (KarolCislowski) [inline server.js:70]: according to documentation `post "/" ` need to create a new one.

File: server.js [hunk 3]
```diff
@@ -1,29 +1,87 @@
...
+  try {
+    const thought = await Thought.updateOne(
+      { _id: req.params.thoughtId },
+      { $inc: { hearts: 1 } }
+    );
+    res.status(201).json(thought);
+  } catch (err) {
+    res.status(401).json({ message: "Heart not added to post", error: err });
+  }
+});
```
Comment (KarolCislowski) [inline server.js:82]: This one should by ` /:thoughtId/like `

File: server.js (hunk 1 above)
Comment (kajsabootin) [reply server.js:51]: We got information from Matilda that "if you have received changes requested due to the names of the endpoints, we will change them to approved" . And if I understand right I haven't been approved regarding my endpoints name.

**Comment by kajsabootin** [comment]
We got information from Matilda that "if you have received changes requested due to the names of the endpoints, we will change them to approved" . And if I understand right I haven't been approved regarding my endpoints name.

**Pull request closed without merging**